core/gui/proxy.config.json

```diff
@@ -14,6 +14,11 @@
     "secure": false,
     "changeOrigin": true
   },
+  "/api/computing-unit": {
```

Comment:
Is this change intentional?

Reply:
Yes, for the frontend proxy in the dev environment